packages/video_player/video_player/lib/video_player.dart

```diff
@@ -400,11 +415,11 @@ class VideoPlayerController extends ValueNotifier<VideoPlayerValue> {
           if (_isDisposed) {
             return;
           }
-          final Duration newPosition = await position;
+          final Duration? newPosition = await position;
           if (_isDisposed) {
             return;
           }
-          _updatePosition(newPosition);
+          _updatePosition(newPosition!);
```

Comment:
can `newPosition` here be null?

Reply:
No, but I changed the code, so it's more readable. The Analyzer can figure out that `newPosition!` can safely be ``newPosition` if you have a null check earlier.